src/cssm.pyx

```diff
@@ -822,7 +822,7 @@ def ddm_flex(np.ndarray[float, ndim = 1] v,
                                                           'n_samples': n_samples,
                                                           'simulator': 'ddm_flex',
                                                           'boundary_fun_type': boundary_fun.__name__,
-                                                          'drift_fun_type': boundary_fun.__name__,
+                                                          'drift_fun_type': drift_fun.__name__,
```

Comment:
is this something that is wrong across the board? 
if yes could you apply that change universally?